### PR TITLE
Added display options, including showing warnings and suggestions.

### PR DESCRIPTION
**Pull request by brennan-nc**
You can now display the password warnings and suggestions returned by zxcvbn.
The password strength meter now supports jQuery UI as an alternative to Bootstrap.
The widget now allows you to validate by password length (min and max).
The project is now tested and working in IE8. Some polyfills were added for this.
Debounce is now used on the keyup event to have it trigger less frequently.
The password strength and confirmation widgets now respect PasswordInput's render_value.
Git ignored PyCharm's idea files directory.

### CODE REVIEW

**Comment by logan-ncc** [comment]
Screenshots?

File: django_password_strength/static/django_password_strength/js/password_strength.js [hunk 1]
```diff
@@ -12,77 +12,177 @@
...
+                        self.hide_progress_bar_jquery_ui(password_strength_bar_jquery_ui);
+                    }
+                    password_strength_info.addClass("hidden");
+                    password_suggestions_list.addClass("hidden");
+                    password_warning_info.addClass("hidden");
                 }
-                self.match_passwords($(this));
-            });
+                self.match_passwords( $(this) );
+            }));
```
Comment (logan-ncc) [inline django_password_strength/static/django_password_strength/js/password_strength.js]: Where did this extra bracket come from?

File: django_password_strength/static/django_password_strength/js/password_strength.js [hunk 2]
```diff
@@ -12,77 +12,177 @@
...
+            password_strength_bar.width('0%').attr('aria-valuenow', 0);
+        },
+
+        update_progress_bar_jquery_ui: function(password_strength_bar, result_zxcvbn) {
+            var color = "#00B200"; // Green
+
+            if( result_zxcvbn.score < this.min_score() ) {
+                color = "#FF0000"; // Red
+            } else if ( result_zxcvbn.score == this.min_score() ) {
+                color = "#FFAE19"; // Orange
```
Comment (logan-ncc) [inline django_password_strength/static/django_password_strength/js/password_strength.js]: Make this a config option?

**Comment by brennan-nc** [comment]
@logan-ncc I didn't want to do screenshots twice, and I wanted them to be part of the release notes for HSMS, so I decided I'd do those in the PR into HSMS.

**Comment by logan-ncc** [comment]
+1

**Comment by Terhands** [comment]
+1 for the code - but seeing the widget would also be nice

File: README.md [hunk 3]
```diff
@@ -25,21 +34,26 @@ There are currently no translations already available, but all the text is trans
 
 For the javascript translations be sure to add the javascript translation catalog [provided by django](https://docs.djangoproject.com/en/1.7/topics/i18n/translation/#using-the-javascript-translation-catalog) or use something like [django-statici18n](https://github.com/zyegfryed/django-statici18n) for a static version of the catalog. If you don't want translations you don't have to add the catalog to your page.
 
-### Example:
+### Simple Example:
+
+The simplest use case is specifying not specifying the settings parameter for the PasswordStrengthInput widget. Note
```
Comment (kumaran-nc) [inline README.md]: "specifying not specifying" seems like a typo.

**Comment by kumaran-nc** [comment]
+1

**Pull request merged**